src/serialize_sequence.cc

```diff
@@ -136,11 +136,35 @@ void serialize_step(const std::filesystem::path& path, const Step& step)
     stream << step; // RAII closes the stream (let the destructor do the job)
 }
 
-void serialize_sequence(const std::filesystem::path& path, const Sequence& sequence)
+std::ostream& operator<<(std::ostream& stream, const Sequence& sequence)
+{
+    stream << "step setup: [[[[[\n";
```

Comment:
I would propose to simplify things a bit for now: Maybe instead of `sequence.txt` we should just have a `sequence.lua` whose entire content is the step setup script, and which always exists (and if it does not, we assume the script is empty). If we add a different language later on, we can easily add a `sequence.py` next to it.

The `require` might be a good idea, but I think it is too early for that. We can add something like that later on if we start thinking about libraries in earnest.